package.json

```
@@ -6,7 +6,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "ocaml": ">= 4.12.0 < 4.13.0",
-    "@grain/libbinaryen": "^101.0.1",
+    "@grain/libbinaryen": "^102.0.0",
```

```suggestion
    "@grain/libbinaryen": "^102.0.1",
```
This is published. I'll get them updated together when opam is published